jax/__init__.py

```diff
@@ -35,6 +35,8 @@
 from jax import config as _config_module
 del _config_module
 
+from jax._src.basearray import Array as Array
```

Comment:
Do we want to export this right now?

Reply:
As I understood it, the goal of this was to expose a working prototype of `jax.Array` so we could migrate users to it before landing the new experimental `Array`, rather than having to change their code twice. So yes, we'll need to export this now.

Reply:
Ahh right! SGTM!